feature-libs/qualtrics/qualtrics.module.ts

```diff
@@ -1,18 +1,7 @@
 import { NgModule } from '@angular/core';
-import { provideDefaultConfig } from '@spartacus/core';
-import {
-  QualtricsComponentsModule,
-  QualtricsConfig,
-} from '@spartacus/qualtrics/components';
+import { QualtricsComponentsModule } from '@spartacus/qualtrics/components';
 
 @NgModule({
   imports: [QualtricsComponentsModule],
-  providers: [
```

Comment:
We still need to provide the default configuration.
Did you provide the `defaultQualtricsConfig` somewhere?

Reply:
yup it was in here to set the default before https://github.com/SAP/spartacus/pull/10578/files#diff-dfdf4829ba4be5c9acbf75828d6d16bd73182a6eebf193df9204db8c5b2d8af4R21

and override it in this module when it was empty